modules/social_features/social_swiftmail/social_swiftmail.module

```diff
@@ -81,8 +81,12 @@ function social_swiftmail_preprocess_swiftmailer(array &$variables) {
     $variables['footer'] = \Drupal::service('renderer')->renderRoot($footer_markup);
   }
 
+  $user = '';
```

Comment:
@volodymyr-sydor-lemberg I understand why it is needed, but think NULL or FALSE would be better here.